### DISCUSSION TITLE
How can get path from inject Get $get in Repeater

### DISCUSSION THREAD

**Discussion by xartem** [filamentphp/filament#9381] [Help] [answered]
Tell me please, how can i get path from inject Get $get in Repeater
<img width="935" alt="image" src="https://github.com/filamentphp/filament/assets/13962589/9c2750bd-9feb-476d-acb6-7a9ab6b69335">
Thanx

**Comment by leandrocfe** [comment, marked as answer]
[This?](https://filamentphp.com/docs/3.x/forms/fields/repeater/#using-get-to-access-parent-field-values)

**Reply by xartem** [reply]
Yes, thanks
